lib/praxis/api_general_info.rb

```diff
@@ -118,6 +118,14 @@ def describe
       hash
     end
 
+    def supported_request_handlers
```

Comment:
same here, these hardcoded should come from the new consumes/produces APIDefinition Info...